hack/release.sh

```diff
@@ -32,7 +32,9 @@ function build_release() {
   GOOS=darwin GOARCH=amd64 go build -mod=vendor -ldflags "${ld_flags}" -o ./kn-darwin-amd64 ./cmd/...
   echo "🚧 🎠 Building for Windows"
   GOOS=windows GOARCH=amd64 go build -mod=vendor -ldflags "${ld_flags}" -o ./kn-windows-amd64.exe ./cmd/...
-  ARTIFACTS_TO_PUBLISH="kn-darwin-amd64 kn-linux-amd64 kn-windows-amd64.exe"
+  echo "🚧 🐳 Building the container image"
+  ko resolve ${KO_FLAGS} -f config/ > kn-image-location.yaml
```

Comment:
Does that now assume `ko` present and the default container image builder? If that's a widely held assumption in Knative then cool. I'd assume some folks would want to use `docker`?

Reply:
Yes, ko is used in all other repos. If you change to docker it will break the release process. Also, release.sh will be hardly used locally, and mostly by the release jobs.